src/ol/source/wmts.js

```diff
@@ -292,6 +292,7 @@ ol.source.WMTS.prototype.updateDimensions = function(dimensions) {
  *  - style - {string} The name of the style
  *  - format - {string} Image format for the layer. Default is the first
  *       format returned in the GetCapabilities response.
+ *  - crossOrigin - {string} Cross origin.
```

Comment:
The type should be the same as the `crossOrigin` option for the WMTS layer: `{string|null|undefined}`